gui/default/index.html

```diff
@@ -344,118 +344,140 @@ <h3 class="panel-title">
-            <p class="text-info" translate>
-              Current domain does not match WebAuthn configuration.
-            </p>
-            <p>
-              <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>
-                Syncthing Documentation: Customizing the GUI address
-              </a>
-            </p>
-          </div>
-          <p class="text-info" ng-if="isLocationInsecure()" translate>
-            WebAuthn requires HTTPS.
-          </p>
-          <p class="text-danger" ng-if="webauthn.errors.notRegistered" translate>
-            This device or security key is not registered. Please try a different one.
-          </p>
-          <p class="text-info" ng-if="webauthn.errors.aborted" translate>
-            Authentication aborted by user.
-          </p>
-          <p class="text-danger" ng-if="webauthn.errors.notAllowed" translate>
-            Authentication failed, please try again.
-          </p>
-          <div ng-if="webauthn.errors.securityError">
-            <p class="text-danger" translate>
-              Initialization blocked by browser. Please make sure Syncthing is correctly configured.
-            </p>
-            <p>
-              <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>
-                Syncthing Documentation: Customizing the GUI address
-              </a>
-            </p>
-          </div>
-          <p class="text-danger" ng-if="webauthn.errors.initFailed" translate>
-            Initialization failed, see Syncthing logs for details.
-          </p>
-          <p class="text-danger" ng-if="webauthn.errors.authenticationFailed" translate>
-            Authentication failed, see Syncthing logs for details.
-          </p>
-          <p class="text-info" ng-if="webauthn.errors.noCredentials" translate>
-            User has not configured any WebAuthn credentials.
-          </p>
-          <p class="text-danger" ng-if="webauthn.errors.uvRequired" translate>
-            User verification is required, but was not performed.
-          </p>
+          <div class="panel panel-default">
+            <button class="btn panel-heading" data-toggle="collapse" data-parent="#login-options" data-target="#login-option-webauthn" aria-expanded="{{isDefaultLoginOption('webauthn') ? 'true' : 'false'}}">
+              <h4 class="panel-title">
+                <div class="panel-title-text" translate>WebAuthn (Passkey)</div>
+              </h4>
+            </button>
+            <div id="login-option-webauthn" class="panel-collapse collapse {{isDefaultLoginOption('webauthn') ? 'in' : ''}}">
+              <div class="panel-body">
+                <div class="form-group">
+                  <div class="col-md-6">
+                    <label class="stay-logged-in">
+                      <input type="checkbox" ng-model="login.stayLoggedIn" ng-disabled="login.inProgress || isWebauthnLoginDisabled()">&nbsp;<span translate>Stay logged in</span>
```

Comment:
Actually, I find it surprising that this checkbox is locked.  Since it's synchronized with the other one, I see no reason to block its usage just because the login method is unavailable.

Reply:
Ok, fixed in abab9e91cd963a49cdc9916b5ca93d41c9d9b89d and 292a9edced589b5d1e01049646515fcc623a450e.